### ISSUE TITLE
Ndex2 2.0 release breaks integration

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#654]
The `ndex2` package was just released (https://pypi.org/project/ndex2/2.0.0/) and it breaks integration with `indra.databases.ndex_client`:
```
======================================================================
ERROR: Failure: ImportError (No module named 'ndex2.niceCXNetwork')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 697, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/travis/build/bgyori/indra/indra/tests/test_eidos.py", line 8, in <module>
    from indra.assemblers.cx import CxAssembler
  File "/home/travis/build/bgyori/indra/indra/assemblers/cx/__init__.py", line 1, in <module>
    from .assembler import CxAssembler
  File "/home/travis/build/bgyori/indra/indra/assemblers/cx/assembler.py", line 10, in <module>
    from indra.databases import context_client, ndex_client, get_identifiers_url
  File "/home/travis/build/bgyori/indra/indra/databases/ndex_client.py", line 10, in <module>
    from ndex2.niceCXNetwork import NiceCXNetwork
ImportError: No module named 'ndex2.niceCXNetwork'
----------------------------------------------------------------------
```

**Comment by bgyori** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Look into what changed in ndex2 v 2.0 and assess difficulty of updating INDRA to work with it](https://trello.com/c/rtJIOXsZ/29-look-into-what-changed-in-ndex2-v-20-and-assess-difficulty-of-updating-indra-to-work-with-it)

**Comment by bgyori** [comment]
Yes, temporarily, we fixed the required version of ndex2 to 1.2.0.58 (see https://github.com/sorgerlab/indra/blob/master/setup.py#L24) which works with INDRA.

**Comment by bgyori** [comment]
Fixed in #696

**Comment by bgyori** [comment]
Reopening because importing ndex2 2.0.0.7 errors in INDRA unless extra dependencies (neo4jrestclient) are added. Will revert to ndex2==1.2.0.58 until a new version is released that doesn't require that dependency.

**Comment by bgyori** [comment]
Fixed in #702
